src/aws/osml/tile_server/viewpoint/worker.py

```diff
@@ -42,26 +48,26 @@ class SupplementaryFileType(str, AutoLowerStringEnum):
 class ViewpointWorker(Thread):
     def __init__(
         self,
-        viewpoint_request_queue: ViewpointRequestQueue,
+        aws_sqs: ServiceResource,
         aws_s3: ServiceResource,
-        viewpoint_database: ViewpointStatusTable,
+        aws_ddb: ServiceResource,
         logger: Logger = logging.getLogger(__name__),
     ):
```

Comment:
Missing return type (`-> None`) since other `__init__` has this.

Reply:
fixed